ComponentKit/Core/Scope/CKDetectComponentScopeCollisions.mm

```diff
@@ -31,9 +32,69 @@
 }
 #pragma clang diagnostic pop
 
-void CKDetectComponentScopeCollisions(const CKComponentLayout &layout)
+static CKComponent * findCollisionComponent(CKComponent *component,
```

Comment:
Nit: There shouldn't be a space after the asterisk.

I'd also consider renaming `component` to make it a bit more obvious what component it is. If the purpose of this function is to find the component that the component scope is colliding with the name should reflect that somehow. Maybe `collidingComponent` and the function could be named `completeComponentScopeCollisionPair`?